lib/schema/tablebuilder.js

```diff
@@ -46,15 +46,29 @@ class TableBuilder {
   }
 
   // The "timestamps" call is really just sets the `created_at` and `updated_at` columns.
-  timestamps() {
-    const method = arguments[0] === true ? 'timestamp' : 'datetime';
-    const createdAt = this[method]('created_at');
-    const updatedAt = this[method]('updated_at');
-    if (arguments[1] === true) {
-      const now = this.client.raw('CURRENT_TIMESTAMP');
-      createdAt.notNullable().defaultTo(now);
-      updatedAt.notNullable().defaultTo(now);
+
+  timestamps(param1, param2, useCamelCase) {
+    const camel = useCamelCase === true;
+    const method = param1 === true ? 'timestamp' : 'datetime';
+    if (camel) {
+      const createdAt = this[method].camelCase('created_at');
+      const updatedAt = this[method].camelCase('updated_at');
+      if (param2 === true) {
+        const now = this.client.raw('CURRENT_TIMESTAMP');
+        createdAt.notNullable().defaultTo(now);
+        updatedAt.notNullable().defaultTo(now);
+      }
     }
+    {
```

Comment:
This is too much duplication for no good reason. The only different part is the name of the fields, and you can actually camelCase them manually